### PR TITLE
update tr.js

### PR DESCRIPTION
**Pull request by lelprodude**
translated a bunch of stuff from all over the document that wasnt translated yet
if its okay can i also make a pull request after this to add my name to the list of contributors

### CODE REVIEW

**Comment by mty22** [comment]
@SecretMember can you have a look at this one please? :)

**Comment by SecretMember** [comment]
I have genuinely no idea why it did not save my progress, I remember sending a pull request. It is looking good whatsoever. I see no problem in adding you to the contributors list.

**Comment by lelprodude** [comment]
i just decided to see if i missed anything and just translated a few more stuff, and fixed a couple things i missed like translating the name of the taunt but not the word taunt itself
if i ever have the time ill compare everything line for line from the english folder cause why not

File: tr.js [hunk 1]
```diff
@@ -838,7 +838,7 @@ stats.crouches Çömelmeler
 stats.sprays Sıkılan Spreyler
 stats.kpd Yardımcı Raporlar
 windows.host.default.settings Varsayılan ayarları açmak oyuncu ilerlemesini ve yüksek skorlarını depolar
-windows.host.default.commands *Kurucu ayarları şu modlarda kapalıdır: /noclip /god /smite vb.
```
[the remark this answers is not in the document]
Comment (lelprodude) [reply tr.js:841]: oh okay my bad

**Pull request merged**